index.html

```diff
@@ -163,37 +174,123 @@ <h2 class="workstitle">Multi-Post Stories</h2>
+                            <li class="javascript">Javascript</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="viewProject">
+                        See Project
+                    </button>
+                </div>
+            </li>
+        </ul>
+        <ul class="card6">
+            <li class="project">
+                <img class="image 2" src="desk/SnapshootPortfolio6.png" alt="Snapshot">
+                <div class="description">
+                    <h2 class="workstitle">Facebook 360</h2>
+                    <div class="info">
+                        <p class="subTitle">FACEBOOK</p>
+                        <ul class="infoContainer">
+                            <li class="role">Full Stack Dev</li>
+                            <li class="year">2015</li>
+                        </ul>
+                    </div>
+                    <p class="projectDescription">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+                    </p>
+                    <div class="tags">
+                        <ul class="tagsFlex">
+                            <li class="html">HTML</li>
+                            <li class="ruby">Ruby on rails</li>
+                            <li class="css">CSS</li>
+                            <li class="javascript">Javascript</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="viewProject">
+                        See Project
+                    </button>
+                </div>
+            </li>
+        </ul>
+        <ul class="card7">
+            <li class="project">
+                <img class="imageReverse" src="desk/SnapshootPortfolio7.png" alt="Snapshot">
+                <div class="descriptionReverse">
+                    <h2 class="workstitle">Uber Navigation</h2>
+                    <div class="info">
+                        <p class="subTitle">UBER</p>
+                        <ul class="infoContainer">
+                            <li class="role">Lead Developer</li>
+                            <li class="year">2018</li>
+                        </ul>
+                    </div>
+                    <p class="projectDescription">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+                    </p>
+                    <div class="tags">
+                        <ul class="tagsFlex">
+                            <li class="html">HTML</li>
+                            <li class="css">CSS</li>
+                            <li class="javascript">Javascript</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="viewProject">
+                        See Project
+                    </button>
+                </div>
+            </li>
+        </ul>
```

Comment:
- [OPTIONAL] Kindly try to wrap all projects inside a single `ul` tag and each project `li` tag, not each project in a separate `ul` tag for semantic purposes.
- [OPTIONAL] Kindly try to match spacing in left and right for each project in mobile version to match the Figma design.

| Your design | Figma design |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/155770164-0b82fc40-a148-4e57-a4cf-394eee5db27a.png) | ![image](https://user-images.githubusercontent.com/24703171/155770288-13de2cdc-30e1-440f-94dc-68a101e13019.png) |